javascript/javascript_sdk/src/index.js

```diff
@@ -1,77 +1,42 @@
-'use strict';
-
-// Arg parsing
 const yargs = require('yargs');
-
-// Message receiver
-const receiver = require('./message_receiver');
-
-// Parsing
-const parsing = require('./parse_output');
-
-module.exports = {
-  run,
-  receiver,
-  parsing,
-};
+const output_sel = require('./output_selector.js')
```

Comment:
can we make this as an example like the previous javascript has?

Reply:
could you elaborate on this more? I am bit confused

Reply:
As I know, the previous javascript has 2 folders. One is javascript_sdk and the other is example.
I think index.js and output_selector does not seems to be javascript_sdk itself. it's more likely example code.
So, I'm suggestion to split index.js and output_selector as one of example.

Reply:
Got it